Images_And_Graphics_style.css_v1

```diff
@@ -354,18 +413,18 @@ ul li:hover ul {
 .Logo2, .Video1, .Image2 .Image6 .Image7 {
 	margin-top: 210px;
 }
-
 .Image3, .Image4 {
 	margin-top: -900px;
 	text-align: center;
 }
 
-.Image5 {
-	margin-left: 200px;
-	margin-top: -390px;
-	margin-bottom: 400px;
+.TawaLogo {
+  position: absolute;
+  top: 20px; /* Adjust this value to position the logo from the top */
+  left: 20px; /* Adjust this value to position the logo from the left */

```

Comment:
I added top 20px this is so the logo is on the top then added left 20px to the left as the logo is suppose to be on the top left.